Frontend/ui-driver/src/Screens/RegistrationScreen/View.purs

```diff
@@ -475,6 +480,7 @@ listItem push item state =
       compStatusImg state item = 
         case getStatus item.stage state of
           ST.COMPLETED -> fetchImage COMMON_ASSET "ny_ic_green_tick"
+          ST.MANUAL_VERIFICATION_REQUIRED -> fetchImage COMMON_ASSET "ny_ic_green_tick"
```

Comment:
Same here what if the manual_verification_required default status is false, shouldn't we show a different color then?